package.json

```diff
@@ -14,17 +14,18 @@
     "@babel/preset-react": "^7.9.4",
     "@babel/preset-typescript": "^7.9.0",
     "@babel/runtime-corejs3": "^7.9.2",
+    "@testing-library/jest-dom": "5.10.0",
+    "@testing-library/react": "10.2.1",
+    "@testing-library/user-event": "^12.0.11",
```

Comment:
This one is new, to be able to type into form fields.
The other lines were just reordered when this was installed

Reply:
Nice one. Didn't know that one existed!